consensus/polybft/fsm.go

```diff
@@ -172,6 +176,24 @@ func (f *fsm) BuildProposal(currentRound uint64) ([]byte, error) {
 		}
 
 		extra.Validators = f.newValidatorsDelta
+
+		if f.config.IsBridgeEnabled() && !f.newValidatorsDelta.IsEmpty() {
```

Comment:
Why not just have a function like `applyBridgeBatchTx`, where you will encapsulate all this logic?